### PR TITLE
to read the file word wise

### PR DESCRIPTION
**Pull request by Shalini-jk**

### CODE REVIEW

File: Sorting/BinarySearch.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+print('\n')
+
+x=int(input('Enter number to search '))
+f=0
+S=0
+E=len(num)-1
+
+while S<=E:
+
+    M=(S+E)//2
```
Comment (anurag-kqi) [inline Sorting/BinarySearch.py:15]: why // two times?

File: Sorting/BinarySearch.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
+num=[2,5,9,12,17,37,86]
+
+print('List: ',end='')
+for n in num:
+    print(n,end=' ')
+print('\n')
+
+x=int(input('Enter number to search '))
```
Comment (anurag-kqi) [inline Sorting/BinarySearch.py:8]: Need spaces around = like x = int(...)

File: Sorting/BinarySearch.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+f=0
+S=0
+E=len(num)-1
+
+while S<=E:
+
+    M=(S+E)//2
+
+    if x==num[M]:
```
Comment (anurag-kqi) [inline Sorting/BinarySearch.py:17]: space x == num[M]

File: Sorting/BinarySearch.py [hunk 4]
```diff
@@ -0,0 +1,29 @@
+num=[2,5,9,12,17,37,86]
+
```
Comment (anurag-kqi) [inline Sorting/BinarySearch.py:2]: Add a check that input array is sorted, otherwise return an error.

File: Sorting/BinarySearch.py [hunk 5]
```diff
@@ -0,0 +1,29 @@
+num=[2,5,9,12,17,37,86]
```
Comment (anurag-kqi) [inline Sorting/BinarySearch.py:1]: space after , like num = [2, 5, 9 ]

File: Sorting/BinarySearch.py [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+print('List: ',end='')
+for n in num:
+    print(n,end=' ')
+print('\n')
+
+x=int(input('Enter number to search '))
+f=0
+S=0
+E=len(num)-1
+
```
Comment (anurag-kqi) [inline Sorting/BinarySearch.py:12]: Explain the algo.

File: Sorting/BubbleSort.py [hunk 7]
```diff
@@ -0,0 +1,18 @@
+nums = []
+print("Enter the Size of List: ")
+tot = int(input())
+
+print("Enter " + str(tot) + " Numbers: ")
+for i in range(tot):
+    nums.insert(i, int(input()))
+
+for i in range(tot-1):
```
Comment (anurag-kqi) [inline Sorting/BubbleSort.py:9]: Explain the algo.

How did you test this program?

What is the complexity of this algo?

File: Sorting/LinearSearch.py [hunk 8]
```diff
@@ -0,0 +1,12 @@
...
+arr = []
+for i in range(10):
+  arr.insert(i, int(input()))
+print("Enter the Number to Search: ")
+num = int(input())
+for i in range(10):
+  if num==arr[i]:
+    index = i
+    break
+print("\n Number Found at Index Number: ")
```
Comment (anurag-kqi) [inline Sorting/LinearSearch.py:11]: What about errro case, when given number is not in the array.

File: Sorting/SelectionSort.py [hunk 9]
```diff
@@ -0,0 +1,15 @@
+Array = [5,6,4,2,6,1,9,8,20,7]
+arry1 = Array
+print(arry1)
+for i in range(len(Array)):
```
Comment (anurag-kqi) [inline Sorting/SelectionSort.py:4]: Explain the algo. 

What i the complexity of this algo,

**Review by anurag-kqi — COMMENTED** [review]
Please make all the required changes.

**Comment by Shalini-jk** [comment]
sir i used // so that the result will round off to the nearest whole number
if the result will be in decimal

**Comment by Shalini-jk** [comment]
okay sir

**Pull request merged**